### DISCUSSION TITLE
Frontend

### DISCUSSION THREAD

**Discussion by Tim-Schwalbe** [kubescape/helm-charts#354] [Ideas]
Is there any Open-Source Frontend available?
It feels a little hard to use otherwise. 
If not, how are you guys using this tool? Just manually scaning through the vulnerabilties with kubectl?

**Comment by alegrey91** [comment]
Hello @Tim-Schwalbe, in order to better visualise information retrieved by kubescape, you should use our platform at: cloud.armosec.io.
You have to create an account and install the helm-chart with your customer uid and access key in order to receive the data (everything is explained in the platform).
The platform is free up to 10 working nodes.
Hope I replied to your question. If you want to know more, feel free to ask :)

**Comment by Oshratn** [comment]
An alternative available now, is to export your data to Grafana dashboard you build. Forr the near future, we are building something for Kubescape 3.1 (see the end of this [blog post](https://kubescape.io/blog/2023/09/19/introducing-kubescape-3/#capabilities-based-in-cluster-installation)), so either watch this space, or you can help with contirbutions of ideas or code.

In any case, I would love to hear what you ultimately landed on and found useful.

**Comment by Liammarwood** [comment]
Any updates on a UI for Kubescape to visualise the reports?

**Reply by Tim-Schwalbe** [reply]
I used this for a while, but now it's not free of charge anymore, so I am looking for alternatives also.
But it's a really nice tool, if you can effort I would recommend it. 
https://cloud.armosec.io

**Reply by Oshratn** [reply]
Yes, [the commercial product has been detached from the open source offering](https://www.armosec.io/blog/armo-platform-vs-kubescape/).
That being said, Kubescape 3.0 brought a [very usable results page on the CLI](https://kubescape.io/blog/2023/10/01/kubescape-3-cli-improvements/).
In addition, a community member recently created a [headlamp plug-in for Kubescape](https://kubescape.io/blog/2023/10/01/kubescape-3-cli-improvements/). It is in alfa, and not production ready, but may be a good starting point for you.
Lastly, an LFX mentee for Kubescape is working on a [Backstage plug-in for Kubescape](https://github.com/kubescape/backstage-plugin). So, if Backsatge is something you are looking at, it could help as well.
